examples/how_to/custom/react/children.py

```diff
@@ -6,8 +6,8 @@ class Example(ReactComponent):
     children = Children()
 
     _esm = """
-    export function render({ children }) {
-      return <div>{children.children}</div>
+    export function render({ model }) {
+      return <div>{model.get_child("children")}</div>
```

Comment:
Have you considered having a separate `get_children` method. Or even just using `model.get("children")`?

Reply:
Yes, haven't strongly come down on what I think here. Having two methods is kind of a shame but I also agree that a list returned by `get_child` is weird.

What I did want to make sure of was that I wanted to align `AnyWidget` and our implementation here. So `.get` is not an option because in the AnyWidget case `model.get(<child-traitlet-name>)` must return the Widget model instance not the rendered child view, DOMElement or React Component.